docs/reusable/media/_index.md

```diff
@@ -0,0 +1,1064 @@
+
+#### Events for data quality
+
+##### Buffer start
+
+Tracks a media player buffering start event fired when the player goes into the buffering state and begins to buffer content.
+
+The tracker will calculate the time since this event until either the buffer end event or play event or a change in playback position and add the duration to the `timeBuffering` property in the session entity.
+
+<>{(props.tracker == 'js-tag') && (<CodeBlock language="javascript">
+{`window.snowplow('trackMediaBufferStart', { id });`}
+</CodeBlock>)}</>
+
+<>{(props.tracker == 'js-browser') && (<CodeBlock language="javascript">
+{`trackMediaBufferStart({ id });`}
+</CodeBlock>)}</>
+
+*Schema:*
+`iglu:com.snowplowanalytics.snowplow.media/buffer_start_event/jsonschema/1-0-0`.
+
+##### Buffer end
+
+Tracks a media player buffering end event fired when the the player finishes buffering content and resumes playback.
+
+<>{(props.tracker == 'js-tag') && (<CodeBlock language="javascript">
+{`window.snowplow('trackMediaBufferEnd', { id });`}
+</CodeBlock>)}</>
+
+<>{(props.tracker == 'js-browser') && (<CodeBlock language="javascript">
+{`trackMediaBufferEnd({ id });`}
+</CodeBlock>)}</>
+
+*Schema:*
+`iglu:com.snowplowanalytics.snowplow.media/buffer_end_event/jsonschema/1-0-0`.
+
+##### Quality change
+
+Tracks a media player quality change event tracked when the video playback quality changes.
+
+The event schema has the following properties:
+
+| Request Key | Required | Type/Format | Description |
+| --- | --- | --- | --- |
+| previousQuality | N | string | Quality level before the change (e.g., 1080p) |
+| newQuality | N | string | Quality level after the change (e.g., 1080p) |
+| bitrate | N | integer | The current bitrate in bits per second |
+| framesPerSecond | N | integer | The current number of frames per second |
+| automatic | N | boolean | Whether the change was automatic or triggered by the user |
+
+The `previousQuality` is set automatically based on the last `quality` value in the `player` entity.
+The `newQuality` is passed when tracking the event and is automatically updated in the `player` entity.
+
+<>{(props.tracker == 'js-tag') && (<CodeBlock language="javascript">
+{`window.snowplow('trackMediaQualityChange', {
+    id,
+    newQuality: '1080p',
+    bitrate: 1000,
+    framesPerSecond: 30,
+    automatic: false,
+});`}
+</CodeBlock>)}</>
+
+<>{(props.tracker == 'js-browser') && (<CodeBlock language="javascript">
+{`trackMediaQualityChange({
+    id,
+    newQuality: '1080p',
+    bitrate: 1000,
+    framesPerSecond: 30,
+    automatic: false,
+});`}
+</CodeBlock>)}</>
+
+*Schema:*
+`iglu:com.snowplowanalytics.snowplow.media/quality_change_event/jsonschema/1-0-0`.
+
+##### Error
+
+Tracks a media player error event tracked when the resource could not be loaded due to an error.
+
+The event schema has the following properties:
+
+| Request Key | Required | Type/Format | Description |
+| --- | --- | --- | --- |
```

Comment:
This should include `errorName` (and the examples below too)

Reply:
Thanks, added!